charts/primary-site/templates/cronjobs/garbage-collector.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: garbage-collector
+  labels:
+    app: foxglove-garbage-collector
+spec:
+  schedule: "*/10 * * * *" # every 10 minutes
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 1
+  successfulJobsHistoryLimit: 3
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          volumes:
+            - name: cloud-credentials
+              secret:
+                secretName: gcp-cloud-credential
+                optional: true
+          containers:
+            - name: garbage-collector
+              image: us-central1-docker.pkg.dev/foxglove-images/images/garbage-collector:dcb5b4812e4d0a6054638ea43712f2e7ee5050ab
```

Comment:
let's parameterize the image so it can be defined in the values file at the top level. That way they will always match as well.

Reply:
another thought, is I think we can use the `.Chart.appVersion` here, and then avoid needing to release new updates by modifying anything in values.yaml at all - we'd just bump the chart version (and would need to start tagging the software accordingly)

Reply:
I like the idea of using the app version for this.

Reply:
> another thought, is I think we can use the `.Chart.appVersion` here, and then avoid needing to release new updates by modifying anything in values.yaml at all - we'd just bump the app version (and would need to start tagging the software accordingly)

This is genius! Just added.